btrdbextras/eventproc/eventproc.py

```diff
@@ -211,3 +214,48 @@ def inner(func):
         return func
 
     return inner
+
+def upload_file(file, file_name):
+    """
+    Uploads file to S3. Returns a link to download the file.
+    If the function runs outside of an eventproc handler executing in response to a hook, it will just check the inputs, raise a warning, and return None.
```

Comment:
Super minor - python community is pedantic about line width.  Suggest wrapping around column 80.

Reply:
Same for elsewhere in this file